tools/build_config/extract_herbert_artifact.ps1

```diff
@@ -0,0 +1,25 @@
+. $PSScriptRoot/powershell_helpers.ps1
+
+# Index "-1" here to deal with case where we may have more than one artifact.
+# We always want the last in the list, that will be the latest as they're
+# sorted alphabetically
+$zip_name_filter = "build/Herbert-*"
+try {
+  $zip_file = (Get-Item "$zip_name_filter")[-1]
```

Comment:
does `Get-Item` return an array if there is only one item??

Reply:
No, it doesn't return a `System.Array` like if two items are found - it returns a `System.IO.FileSystemInfo` object, which is apparently also indexable (but only by 0 or -1 - returning itself). So this syntax works